File: WS/Models/IN/ProduitsSearchParameters.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WS.Models.IN
{
    public class ProduitsSearchParameters
    {

        public Int32? Id { set; get; }
        public String Reference { set; get; }
        public String Libelle{ set; get; }


        public Int32? StockMin { set; get; }
        public Int32? StockMax { set; get; }

        public Int32? CategorieId { set; get; }

        public Int32? CommandeId { set; get; }

    }

}

File: Angular/app/Produits/Models/ProduitsSearchParameters.ts
export class ProduitsSearchParameters {

    constructor() {
        this.Id = null;

        this.Libelle = null;
        this.Reference = null;

        this.CategorieId = null;

        this.StockMin = null;
        this.StockMax = null;

        this.CommandeId = null;

        this.NbCommandes = null;

    }

    public Id: number | null;

    public Libelle: string | null;
    public Reference: string | null;

    public CategorieId: number | null;

    public StockMin: number | null;
    public StockMax: number | null;

    public CommandeId: number | null;

    public NbCommandes: number | null;


}